metacells/tools/named.py

```diff
@@ -35,23 +36,30 @@ def find_named_genes(
     This creates a mask of all the genes whose name appears in ``names`` or matches any of the
     ``patterns``. If ``invert`` (default: {invert}), invert the resulting mask.
 
+    If ``name_property`` is specified the mask will be based on a column in the ``adata.var`` DataFrame
```

Comment:
Change "a column in the ``adata.var`` DataFrame" to "this per-variable (gene) property."